### ISSUE TITLE
declare a default notebook which is automatically opened without password data

### ISSUE THREAD

**Issue by SirMossi** [n1try/mininote#48]
The idea behind this feature is, that there could be some standard notes, which should be visible to everyone without having access to the other notebooks.

**Comment by n1try** [comment]
Although I don't see a lot of use cases for such a feature, I agree this would certainly be a nice-to-have.

Should probably be implemented AFTER #14 and #9.

Feel free to contribute 🙂
